templates/candidate/showMyAcceptedApplications.html.twig

```diff
@@ -0,0 +1,35 @@
+{% extends 'dashboard_layout.html.twig' %}
+
+{% block title %}
+	{{ parent() ~ ' | Mes candidatures acceptées' }}
+{% endblock %}
+
+{% block sideNav %}
+	{% include './component/_sideBar_dashboard.html.twig' %}
+{% endblock %}
+
+{% block content %}
+	<div class="container">
+		<div class="d-flex justify-content-center align-items-center gap-3 p-5 rounded text-dark">
+			<svg xmlns="http://www.w3.org/2000/svg" width="60" height="60" fill="currentColor" class="bi bi-briefcase-fill" viewbox="0 0 16 16">
+				<path d="M6.5 1A1.5 1.5 0 0 0 5 2.5V3H1.5A1.5 1.5 0 0 0 0 4.5v1.384l7.614 2.03a1.5 1.5 0 0 0 .772 0L16 5.884V4.5A1.5 1.5 0 0 0 14.5 3H11v-.5A1.5 1.5 0 0 0 9.5 1h-3zm0 1h3a.5.5 0 0 1 .5.5V3H6v-.5a.5.5 0 0 1 .5-.5z"/>
+				<path d="M0 12.5A1.5 1.5 0 0 0 1.5 14h13a1.5 1.5 0 0 0 1.5-1.5V6.85L8.129 8.947a.5.5 0 0 1-.258 0L0 6.85v5.65z"/>
+			</svg>
+			<h1>Mes candidatures acceptées</h1>
+		</div>
+		<hr>
+		<div class="row justify-content-center g-4 mt-2">
+			{% if applications|length > 0 %}
+				{% for application in applications	 %}
```

Comment:
un for/else pltuot ? https://twig.symfony.com/doc/2.x/tags/for.html#the-else-clause